gdk4/sys/src/lib.rs

```diff
@@ -96,7 +96,8 @@ pub const GDK_PAD_BUTTON_RELEASE: GdkEventType = 24;
 pub const GDK_PAD_RING: GdkEventType = 25;
 pub const GDK_PAD_STRIP: GdkEventType = 26;
 pub const GDK_PAD_GROUP_MODE: GdkEventType = 27;
-pub const GDK_EVENT_LAST: GdkEventType = 28;
+pub const GDK_TOUCHPAD_HOLD: GdkEventType = 28;
+pub const GDK_EVENT_LAST: GdkEventType = 29;
```

Comment:
This should probably be ignored no?

Reply:
Probably, but you don't ignore any of these elsewhere either in gtk4-rs from what I can see.

Reply:
Well, should be hidden in the non-sys bindings. In the -sys bindings it can be useful in the same way as in C.

Reply:
We do in GdkMemoryFormat